docs/cpp2/declarations.md

```diff
@@ -14,6 +14,7 @@ All Cpp2 declarations are written as **"_name_ `:` _kind_ `=` _statement_"**.
 
 > Note: When the type is omitted, whitespace does not matter, and writing `#!cpp x: = 0;` or `#!cpp x : = 0;` or `#!cpp x := 0;` or other whitespace is just a stylistic choice. This documentation's style uses the last one, except when there are multiple adjacent declaration lines this style lines up their `:` and `=`.
 
+> Note: The only variation to the above is that 'constexpr' functions are written with `==` instead of `=` (e.g., `#!cpp square: (i: int) == i * i;`).
```

Comment:
Also, the tersest lambda syntax can skip the `=` entirely. But I think for both, Herb might have not added them yet to the docs as they both have contentions in their current form. https://github.com/hsutter/cppfront/discussions/714 for tersest lambda syntax, https://github.com/hsutter/cppfront/discussions/959 for `constexpr` with `==`.